cpu/mips32r2_common/Makefile.include

```diff
@@ -7,6 +7,7 @@ include $(RIOTMAKE)/arch/mips.inc.mk
 export USEMODULE += periph
 export USEMODULE += periph_common
 export USEMODULE += newlib
+export USEMODULE += periph_timer
```

Comment:
- remove the export?
- a comment why the cpu depends always on the periph_timer might be useful (as done so above for other cases)

Reply:
added a comment